yt_dlp/extractor/yle_areena.py

```diff
@@ -0,0 +1,41 @@
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+)
+
+
+class YleAreenaIE(InfoExtractor):
+    _VALID_URL = r'https?://areena\.yle\.fi/(?P<id>[0-9-]+)'
+    _TESTS = [{
+        'url': 'https://areena.yle.fi/1-4371942',
+        'md5': '932edda0ecf5dfd6423804182d32f8ac',
+        'info_dict': {
+            'id': '0_a3tjk92c',
+            'ext': 'mp4',
+            'title': 'K1, J2: Pouchit | Modernit miehet',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'uploader_id': 'ovp@yle.fi',
+            'duration': 1435,
+            'view_count': int,
+            'upload_date': '20181204',
+            'timestamp': 1543916210,
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        video_data_url = f'https://player.api.yle.fi/v1/preview/{video_id}.json?app_id=player_static_prod&app_key=8930d72170e48303cf5f3867780d549b'
+        video_data = self._download_json(video_data_url, video_id)
+        kaltura_id = traverse_obj(video_data, ('data', 'ongoing_ondemand', 'kaltura', 'id'))
+
+        webpage = self._download_webpage(url, video_id)
+        info = self._search_json_ld(webpage, video_id, default={})
+
+        return {
+            '_type': 'url_transparent',
+            'url': f'kaltura:1955031:{kaltura_id}',
+            'ie_key': 'Kaltura',
+            'title': info.get('title'),
+            'thumbnail': traverse_obj(info, ('thumbnails', 0, 'url')),
```

Comment:
shouldnt this be:
```suggestion
            'thumbnails': traverse_obj(info, ('thumbnails', ..., {'url': 'url'})),
```
This will extract all available thumbnails

Reply:
Kaltura exporter will also extract thumbnail (wrong one) as `thumbnail` key. Is there a better way to replace it or not use it while getting all thumbnails from yle?